presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -327,6 +327,27 @@ Approximate Aggregate Functions
     for all ``value``\ s. This function is equivalent to the variant of
     :func:`numeric_histogram` that takes a ``weight``, with a per-item weight of ``1``.
 
+.. function:: approx_most_frequent(buckets, value, capacity) -> map<[same as value], bigint>
+
+    Computes the top frequent values up to ``buckets`` elements approximately.
+    Approximate estimation of the function enables us to pick up the frequent
+    values with less memory. Larger ``capacity`` improves the accuracy of
+    underlying algorithm with sacrificing the memory capacity. The returned
+    value is a map containing the top elements with corresponding estimated
+    frequency.
+
+    ``buckets`` and ``capacity`` must be ``bigint``. ``value`` can be numeric
+    or string type.
+
+    The function uses the stream summary data structure proposed in the
+    following paper.
+
+    .. code-block:: none
+
+        A.Metwalley, D.Agrawl, A.Abbadi, "Efficient computation of frequent
+        and top-k elements in data streams"
+
+
```

Comment:
@mosabua, can you help review this section?

Reply:
Use a link in the sentence instead

```
The function uses the stream summary data structure proposed in the
 paper `Efficient computation of frequent and top-k elements in data streams by A.Metwalley, D.Agrawl, A.Abbadi`_.

```

Reply:
Or if you just want the title to be the link

```
The function uses the stream summary data structure proposed in the paper `Efficient computation of frequent and top-k elements in data streams`_ by A.Metwalley, D.Agrawl and A.Abbadi.

```